doc/cla/individual/mhabboush.md

```diff
@@ -0,0 +1,11 @@
+Saudi Arabia, 2021-06-18
+
+I hereby agree to the terms of the Odoo Individual Contributor License
+Agreement v1.0.
+
+I declare that I am authorized and able to make this agreement and sign this
+declaration.
+
+Signed,
+
+Mahmoud Habboush mhabboush@users.noreply.github.com https://github.com/mhabboush
```

Comment:
The email needs to be `62886470+mhabboush@users.noreply.github.com`, including the digits and the `+`.